rosdep/base.yaml

```diff
@@ -6578,6 +6578,15 @@ usbutils:
     homebrew:
       packages: [mikhailai/misc/usbutils]
   ubuntu: [usbutils]
+util-linux:
+  arch: [util-linux]
+  debian:
+    buster: [util-linux]
+  fedora: [util-linux]
+  gentoo: [util-linux]
+  macports: [util-linux]
+  ubuntu:
+    focal: [util-linux]
```

Comment:
Likewise this is available on all ubuntu distros: https://packages.ubuntu.com/focal/util-linux